Britannia/localization/english/BTWK_cultures_l_english.yml

```diff
@@ -47,53 +47,63 @@
  angle_prefix:0 "Anglo"
  angle_collective_noun:0 "Angles"
  name_list_angle:0 "Angle"
- oldfrisian:0 "Old Frisian"
- oldfrisian_prefix:0 "Friso"
- oldfrisian_collective_noun:0 "Old Frisians"
+ # oldfrisian:0 "Old Frisian"
+ # oldfrisian_prefix:0 "Friso"
+ # oldfrisian_collective_noun:0 "Old Frisians"
  name_list_oldfrisian:0 "Old Frisian"
- # frisian:0 "Frisian"
- # frisian_collective_noun:0 "Frisians"
+ frisian:0 "Frisian"
```

Comment:
These localisations were commented out because they don't actually function here - the vanilla localization will override them. If you want to explicitly override that, you need to put the localisation in localization\english\replace\cultures_replace_BTWK_l_english.yml.

Reply:
Will remove them.